### PR TITLE
Update UMass Lowell data

### PR DESCRIPTION
**Pull request by jeapostrophe**

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -5572,6 +5573,7 @@ Hong Mei,Peking University,http://sei.pku.edu.cn/~meih/index_en.html,b-ZYIwoAAAA
 Hong Q. Ding,University of Texas at Arlington,http://ranger.uta.edu/~chqding,q7FfnjgAAAAJ
 Hong Qin,Stony Brook University,http://www3.cs.stonybrook.edu/~qin,Wpsc65EAAAAJ
 Hong Shen 0001,University of Adelaide,https://cs.adelaide.edu.au/~hong,tBnVrjEAAAAJ
+Hong Yu,University of Massachusetts Lowell,http://bio-nlp.org/,TyXe64wAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:5576]: AFAIK Hong is at UMass Med.

File: csrankings.csv [hunk 2]
```diff
@@ -12833,6 +12837,7 @@ Seyed Ghassem Miremadi,Sharif University of Technology,http://sina.sharif.ac.ir/
 Seyed Hamed Hassani,University of Pennsylvania,https://www.seas.upenn.edu/~hassani,NOSCHOLARPAGE
 Seyed Hassan Mirian,Sharif University of Technology,http://ce.sharif.edu/~mirian,NOSCHOLARPAGE
 Seyed Masoud Sadjadi,Florida International University,http://www.cis.fiu.edu/~sadjadi,NOSCHOLARPAGE
+Seyed Reza Ahmadzadeh,University of Massachusetts Lowell,http://http://www.ahmadzadeh.info/,8sqAOOoAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:12840]: broken URL

File: csrankings.csv (hunk 1 above)
Comment (emeryberger) [reply csrankings.csv:5576]: (and thus cannot be listed for UML)

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 1 above)
Comment (jeapostrophe) [reply csrankings.csv:5576]: She moved to UML in Jan 2018.

https://www.uml.edu/Sciences/computer-science/faculty/Yu-Hong.aspx

File: csrankings.csv (hunk 2 above)
Comment (jeapostrophe) [reply csrankings.csv:12840]: Updated

**Review by emeryberger — APPROVED** [review]

**Pull request merged**